src/overlays/actors/ovl_En_Test/z_en_test.c

```diff
@@ -1,7 +1,7 @@
 /*
  * File: z_en_test.c
  * Overlay: ovl_En_Test
- * Description:
+ * Description: Goron crater mark
```

Comment:
Is this the crater mark made by link-goron doing a ground slam? Is this specific to link-goron or is this used with other gorons as well?

Reply:
Actually, in decompiling En_Fall, I found it's used for a variety of crater marks; one non-Goron example is the Moon's Tear when it slams into the ground. So I'll change the description to better reflect that.